### ISSUE TITLE
Iteration Plan for July 2019

### ISSUE THREAD

**Issue by lukekim** [dapr/dapr#73]
This plan captures our work in **July**. This is a 4-week iteration. We will **ship** our July Update in **early August**.

## Themes

- Improve project testability, supportability, extensibility and ability to contribute
- Unblock the C# Actors support

## Endgame

July 29th, 2019: Endgame begin
August 2nd, 2019: Endgame complete

The endgame details for this iteration are tracked <TBD>

## Plan Items

Below is a summary of the top level plan items.

Legend of annotations:

|Mark | Description|
|-- | --|
|🏃 | work in progress|
|✋ | blocked task|
|💪 | stretch goal for this iteration|
|🔴 | missing issue reference|
|🔵 | more investigation required to remove uncertainty|
|⚫️ | under discussion within the team|

### Spec
- [x] #94 Migrate spec from old org to new org

### Runtime

- [x] #43 Rename assigner to placement area/assigner area/runtime
- [x] #84 Add skeleton release notes file for 0.2.0-alpha
- [x] #79 Design doc and review for decision on separation of state stores & message buses outside of Bindings
- [x] #76 Separate Actions APIs (gRPC & HTTP) implementations from protocols into its own Go pkg
- [x] #83 API versioning in Spec and implementation
- [x] #78 Change the interface of Bindings

### Demo/Sample

- [x] Add Redis.yaml to kubernetes sample area/sample #108
- [x] Update standalone sample to remove complexity area/sample #107
- [x] Add container devops information in samples area/sample #106

### CLI

- [x] #70 Review and adjust CLI versioning to match with runtime
- [x] actionscore/cli - Actions CLI should be named as actions.exe for Windows (14)

### Engineering Systems

- [x] #81 Create Actions Azure subscription
- [ ] #82 ESRP onboarding
- [x] #99 change runtime image in actions.yaml
